src/components/utilsNew/NoteCard.tsx

```diff
@@ -1,8 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import { Card, Text } from "@chakra-ui/react";
 
-export default function NoteCard() {
+interface NoteCardProps {
+  id: number;
+  text: string;
+  deleteCard: any;
```

Comment:
What does deleteCard represent? May want to add comment for future devs (as a best practice) :)